target/ppc/translate/fixedpoint-impl.c.inc

```diff
@@ -106,9 +124,9 @@ static bool do_ldst_quad(DisasContext *ctx, int ra, TCGv tgt_a, TCGv tgt_b,
         }
     } else {
         if (store) {
-            gen_qemu_st64_i64(ctx, tgt_a, ea);
+            tcg_gen_qemu_st_tl(tgt_a, ea, ctx->mem_idx, mop);
```

Comment:
Since this must always be 64 bits of data, shouldn't we use _i64 instead of _tl for these ops?